dist-git.spec

```diff
@@ -1,7 +1,7 @@
 %global selinux_variants mls targeted
 %global selinux_policyver %(%{__sed} -e 's,.*selinux-policy-\\([^/]*\\)/.*,\\1,' /usr/share/selinux/devel/policyhelp || echo 0.0.0)
 %global modulename dist_git
-
+%global installdir /srv
```

Comment:
Mentioned probably elsewhere, but `/srv` is not allowed by packaging guidelines IMO.